test/com/amazon/ion/impl/bin/IonManagedBinaryWriterTest.java

```diff
@@ -253,4 +423,19 @@ public void testNestedEmptyAnnotatedContainer() throws Exception
         writer.stepOut();
         assertValue("{bar: foo::[]}");
     }
+
+    /**
+     * Asserts equivalence of ion data model between two provided data streams.
+     * @param actual represents the serialized data streams when auto-flush is enabled.
+     * @param expected represents the expected data streams.
+     */
+    private void assertEquivalentDataModel(ByteArrayOutputStream actual, ByteArrayOutputStream expected) {
+        IonLoader loader = IonSystemBuilder.standard().build().newLoader();
+        IonDatagram actualDatagram = loader.load(actual.toByteArray());
+        IonDatagram expectedDatagram = loader.load(expected.toByteArray());
+        assertEquals(actualDatagram.size(), expectedDatagram.size());
+        for (int i = 0; i < actualDatagram.size(); i++) {
+            assertTrue(Equivalence.ionEquals(actualDatagram.get(i), expectedDatagram.get(i)));
+        }
```

Comment:
Can you replace this with `assertEquals(expectedDatagram, actualDatagram);`?

Reply:
Thanks for the suggestion. I will replace it in the next commit.